### PR TITLE
Move to signing service v2

### PR DESCRIPTION
**Pull request by onovotny**
Note: you'll find the updated username and password in your lastpass. both should be treated as secure variables in your build system.

### CODE REVIEW

File: build.cake [hunk 1]
```diff
@@ -313,8 +313,13 @@ Task("Sign-Binaries")
...
         throw new InvalidOperationException("Could not resolve signing secret.");
     }
+    // Get the user.
+    var user = EnvironmentVariable("SIGNING_USER");
+    if(string.IsNullOrWhiteSpace(user)) {
+        throw new InvalidOperationException("Could not resolve signing user.");
+    }
 
-    var client = File("./tools/SignClient/tools/SignClient.dll");
+    var client = File("./tools/SignClient/tools/netcoreapp2.0/SignClient.dll");
```
Comment (devlead) [inline build.cake]: Currently we're building using the 1.0.7 SDK(2.0 is on the way just not yet), so I suspect this version requires .NET Core 2.0? So we should use the netcoreapp1.1 moniker instead right?

Comment (onovotny) [reply build.cake]: k

Comment (devlead) [reply build.cake]: Excellent 👍  , I'll take this for a spin.

File: build.cake [hunk 2]
```diff
@@ -329,12 +338,13 @@ Task("Sign-Binaries")
 
         // Build the argument list.
         var arguments = new ProcessArgumentBuilder()
-            .AppendQuoted(MakeAbsolute(client.Path).FullPath)
-            .Append("zip")
+            .AppendQuoted(signClientPath.FullPath)
+            .Append("sign")
```
Comment (onovotny) [inline build.cake]: Yes, thanks missed this one, thanks for catching it!

**Comment by mholo65** [comment]
@onovotny should this repo also be updated https://github.com/cake-build/bakery?

**Comment by onovotny** [comment]
@mholo65 Yes, didn't see that, but should be the same set of changes. Anywhere you are signing with the existing service/config should be moved over.

BTW, the service does support VSIX's if you have any addin's/templates.

**Comment by mholo65** [comment]
@onovotny thanks! I’ll update that one then.

Do we sign VSIX packages? @gep13 @agc93?

**Comment by gep13** [comment]
@mholo65 no, we aren't currently signing VSIX packages.  We probably should.  Both the Visual Studio and Visual Studio Code packages.

**Comment by onovotny** [comment]
Sign all the things! :)

**Comment by devlead** [comment]
@onovotny your changes have been merged, thanks for your contribution 👍

**Comment by gep13** [comment]
@onovotny @devlead was there a linked issue for this PR?

**Pull request merged**